torch/optim/rmsprop.py

```diff
@@ -348,14 +356,6 @@ def _multi_tensor_rmsprop(
 
         grouped_grads = list(grouped_grads)
```

Comment:
I think this conversion to `list` can be removed maybe?

Reply:
good catch!